cachito/web/models.py

```diff
@@ -440,7 +447,29 @@ def from_json(cls, kwargs):
 
         # current_user.is_authenticated is only ever False when auth is disabled
         if current_user.is_authenticated:
-            request_kwargs['user_id'] = current_user.id
+            user_override_username = request_kwargs.pop('user', None)
```

Comment:
What are your thoughts on using `proxy_username` instead of `user_override_username`. I find the latter a bit difficult to comprehend.

Reply:
Good point. I used `submitted_for_username`. How does that sound?